tests/repositories/test_legacy_repository.py

```diff
@@ -477,7 +477,7 @@ def __init__(
 def test_get_200_returns_page(http: type[httpretty.httpretty]) -> None:
     repo = MockHttpRepository({"/foo/": 200}, http)
 
-    assert repo.get_page("foo")
+    _ = repo.get_page("foo")
```

Comment:
Why did you remove `assert`?

Reply:
iirc the API changed here so that this method never returns `None` any more and the assert is therefore always true

(it instead raises an exception to signal failure)